netlify.toml

```diff
@@ -1,5 +1,4 @@
 [build]
-environment = { PYTHON_VERSION = "3.13" }
 base = "."
 publish = "site"
-command = "mkdocs build -f mkdocs-minify.yml"
\ No newline at end of file
+command = "make update && make build-docs"
```

Comment:
@abnerbog we can't use `docker compose` because the netlify build environment does not have docker installed.

Reply:
You did it! Nice!